### PR TITLE
[Update] static page and config hmr [#4]

### PR DESCRIPTION
**Pull request by dinhthi12**
# Related Tickets

- [#Ticket_ID](https://edu-redmine.vn/issues/xxxx)

## WHAT does this PR do?

- 1. Add hmr and setup the application.

## Notes Impacted Areas

- N/A.

## Attachments

- N/A.

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<!-- app/views/layouts/_footer.html.erb -->
+<footer class="bg-secondary text-light">
+  <p>&copy; 2024 My Application. All rights reserved.</p>
+  <!-- Thêm nội dung footer ở đây -->
+</footer>
```
Comment (hoangpt1310) [inline app/views/layouts/_footer.html.erb]: ok'

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+            <li class="notification-item">
+              <i class="bi bi-exclamation-circle text-warning"></i>
+              <div>
+                <h4>Lorem Ipsum</h4>
+                <p>Quae dolorem earum veritatis oditseno</p>
+                <p>30 min. ago</p>
+              </div>
+            </li>
+
```
Comment (hoangpt1310) [inline app/views/layouts/_header.html.erb]: ok

**Comment by hoangpt1310** [comment]
fix

File: Gemfile [hunk 3]
```diff
@@ -9,11 +9,13 @@ gem "rails", "~> 7.1.3", ">= 7.1.3.2"
...
 # Use webpack to manage app-like JavaScript modules in Rails
-gem 'webpacker'
+gem "webpacker"
 
 # Provides the generator settings required for Rails to use Slim
-gem 'slim', '~> 5.2', '>= 5.2.1'
-gem 'slim-rails', '~> 3.6', '>= 3.6.3'
+gem "slim", "~> 5.2", ">= 5.2.1"
+gem "slim-rails", "~> 3.6", ">= 3.6.3"
+
```
Comment (tn0810520) [inline Gemfile:17]: note

File: Gemfile [hunk 4]
```diff
@@ -27,10 +29,10 @@ gem "puma", ">= 5.0"
 # Use JavaScript with ESM import maps [https://github.com/rails/importmap-rails]
 gem "importmap-rails"
 
-# Hotwire's SPA-like page accelerator [https://turbo.hotwired.dev]
```
Comment (tn0810520) [inline Gemfile]: sai

File: app/javascript/packs/application.js [hunk 5]
```diff
@@ -15,8 +15,13 @@
 // const images = require.context('../images', true)
 // const imagePath = (name) => images(name, true)
 // app/javascript/packs/application.js
+import $ from 'jquery';
 import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import "bootstrap-icons/font/bootstrap-icons.css";
+
```
Comment (tn0810520) [inline app/javascript/packs/application.js:26]: ///

**Pull request merged**